RecoEgamma/EgammaTools/interface/EgammaPCAHelper.h

```diff
@@ -28,6 +28,8 @@
 
 class HGCalRecHit;
 
+namespace hgcal {
+
 class EGammaPCAHelper
```

Comment:
If I'm not mistaken, the indentation is off now
the code inside ```namespace hgcal {``` has no extra offset.
IIUC, typical editor settings will lead to any edits inside the scope become inconsistently indented or will move all the code.
Better do it now, at conception.